inputFiles/poromechanicsFractures/PoroElastic_efem-edfm_pennyCrack_smoke.xml

```diff
@@ -0,0 +1,101 @@
+<?xml version="1.0" ?>
+
+<Problem>
+  <Included>
+   <File name="./PoroElastic_efem-edfm_pennyCrack_base.xml"/>
+  </Included>
+  <Solvers
+    gravityVector="{ 0.0, 0.0, 0.0 }">
+    <SinglePhasePoromechanicsEmbeddedFractures
+      name="poroSolver"
+      targetRegions="{ Domain, Fracture }"
+      initialDt="1e-3"
+      fracturesSolverName="fractureMechSolver"
+      flowSolverName="flowSolver"
+      solidSolverName="matrixMechSolver"
+      logLevel="1">
+      <NonlinearSolverParameters
+        maxTimeStepCuts="1"
+        newtonMaxIter= "10"
+        newtonTol="1e-3"/>
+      <LinearSolverParameters
+        solverType="direct"/>        
+    </SinglePhasePoromechanicsEmbeddedFractures>
+
+    <SolidMechanicsEmbeddedFractures
+      name="fractureMechSolver"
+      targetRegions="{ Domain, Fracture }"
+      solidSolverName="matrixMechSolver"
+      fractureRegionName="Fracture"
+      contactRelationName="fractureContact"/>
+
+    <SolidMechanicsLagrangianSSLE
+      name="matrixMechSolver"
+      timeIntegrationOption="QuasiStatic"
+      discretization="FE1"
+      targetRegions="{ Domain }"/>
+
+    <SinglePhaseFVM
+      name="flowSolver"
+      discretization="singlePhaseTPFA"
+      targetRegions="{ Domain, Fracture }"/>
+
+    <EmbeddedSurfaceGenerator
+      name="SurfaceGenerator"
+      discretization="FE1"
+      targetRegions="{ Domain, Fracture }"
+      targetObjects="{FracturePlane}"
+      fractureRegion="Fracture"
+      logLevel="2"
+      mpiCommOrder="1"/>
+  </Solvers>
+
+  <Mesh>
+    <InternalMesh
+      name="mesh1"
+      elementTypes="{ C3D8 }"
+      xCoords="{ 0, 15 }"
+      yCoords="{ 0, 10 }"
+      zCoords="{ -5, 5 }"
+      nx="{ 45 }"
```

Comment:
we should make this mesh a lot smaller for it to become a smoke test.

Reply:
yeah, also, for the benchmark one, I don't think we need to run it until t = 2000, as it reaches steady-state at about t = 50.